test/evergreen_develop.yml

```diff
@@ -52,6 +52,18 @@ buildvariants:
     <<: *ubuntu2004-perf-tests-expansions-template
     collection: AllPerfTestsARM64
 
+- <<: *perf-tasks-template
```

Comment:
This new variant is necessary as you cannot extend the tasks of a templated variant. If you try and add them to the existing ARM testing variant you override the full list.

Reply:
Fair enough. I guess there's no plan to support these new perf tests running in x86 architecture?

Reply:
And with the current variant settings (without "batchtime"), the new perf tests will be scheduled to run for every single commit. Hope to confirm that's the intended effect.

Reply:
>  I guess there's no plan to support these new perf tests running in x86 architecture?

Yes, no plans at this stage.

The batch time is inherited from the template which is 360, am I missing something?

Reply:
Ah sorry, I missed the batchtime setting in the template.